packages/presentation/lib/my_app.dart

```diff
@@ -22,92 +23,149 @@ class MyApp extends StatefulWidget {
+            ),
+          ),
+          home: StreamBuilder<TileWrapper>(
+            stream: bloc.dataStream,
+            builder: (context, result) {
+              final blocData = result.data;
+              final appData = blocData?.data;
+              if (appData is AppData) {
+                return Scaffold(
+                  body: Row(
+                    children: [
+                      Dimens.isWide(context) && appData.showBottomBar
+                          ? NavigationRail(
+                              minWidth: Dimens.size30.w,
+                              backgroundColor: ColorsApplication.colorDrawer,
+                              selectedIndex: appData.selectedTab,
+                              onDestinationSelected: bloc.onSelectedTab,
+                              destinations: [
+                                NavigationRailDestination(
+                                  icon: SvgPicture.asset(ImagesUtils.movieReel),
+                                  label: const Text(_emptyString),
+                                  selectedIcon: SvgPicture.asset(
+                                    ImagesUtils.movieReel,
+                                    color: ColorsApplication.colorSelectedItem,
+                                  ),
+                                ),
+                                NavigationRailDestination(
+                                  icon:
+                                      SvgPicture.asset(ImagesUtils.eventTicket),
+                                  label: const Text(_emptyString),
+                                  selectedIcon: SvgPicture.asset(
+                                    ImagesUtils.eventTicket,
+                                    color: ColorsApplication.colorSelectedItem,
+                                  ),
+                                ),
+                                NavigationRailDestination(
+                                  icon: SvgPicture.asset(ImagesUtils.alarm),
+                                  label: const Text(_emptyString),
+                                  selectedIcon: SvgPicture.asset(
+                                    ImagesUtils.alarm,
+                                    color: ColorsApplication.colorSelectedItem,
+                                  ),
+                                ),
+                                NavigationRailDestination(
+                                  icon: SvgPicture.asset(ImagesUtils.single),
+                                  label: const Text(_emptyString),
+                                  selectedIcon: SvgPicture.asset(
+                                    ImagesUtils.single,
+                                    color: ColorsApplication.colorSelectedItem,
+                                  ),
+                                ),
+                              ],
+                            )
+                          : const SizedBox.shrink(),
+                      Expanded(
```

Comment:
why Expanded?

Reply:
https://api.flutter.dev/flutter/widgets/Navigator-class.html
https://api.flutter.dev/flutter/widgets/Expanded-class.html

after reading this, you will understand that the navigator manages its children and tries to take the maximum space, for this, expanded comes to our aid